changelog/7758.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed incorrect ``reference_date`` for SDO/AIA and SDO/HMI data to use the ``T_OBS`` keyword instead of the ``DATE-OBS`` keyword.
```

Comment:
Because of the heavy use of sunpy for AIA imagery, it might be helpful to have a second sentence that says, "This correction is typically negligible: for AIA images, the location of SDO has been shifted up to a second in time, which corresponds to a shift in heliographic longitude by only 4 milliarcseconds."